charts/seed-monitoring/charts/prometheus/templates/config.yaml

```diff
@@ -472,3 +472,40 @@ data:
       - target_label: __address__
         replacement: 127.0.0.1:9115
         action: replace
+
+{{- range $i := until 10 }}
+    - job_name: blackbox-exporter-k8s-service-check-{{ $i }}
+      params:
+        module:
+        - http_apiserver
+      scrape_timeout: 10s
```

Comment:
This timeout should be lowered

Reply:
> With a loaded API server we should probably allow 3-5 seconds on single connection attempts

This is not correct - the 99 percentile for requests is about 0.125s on very loaded cluster:

![screenshot 2019-01-08 at 17 31 00](https://user-images.githubusercontent.com/33343952/50840496-54fff600-136b-11e9-9d07-8885d2751ba1.png)

I see that there are 2 use cases 

- test general connectivity with low timeout 
- test with long timeout